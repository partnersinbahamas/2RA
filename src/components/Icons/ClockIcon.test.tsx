import React from 'react';
import { render } from '@testing-library/react';

import ClockIcon from './ClockIcon';

describe('ClockIcon', () => {
  it('renders small', () => {
    const { container } = render(<ClockIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<ClockIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<ClockIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<ClockIcon />);
    expect(container).toMatchSnapshot();
  });
});
