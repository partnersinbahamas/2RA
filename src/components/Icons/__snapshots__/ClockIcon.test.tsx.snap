// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClockIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M10 0C15.516 0 20 4.484 20 10C20 15.516 15.516 20 10 20C4.484 20 0 15.516 0 10C0 4.484 4.484 0 10 0ZM10 2.019C8.9517 2.01821 7.91353 2.22411 6.94487 2.62491C5.97621 3.02571 5.09608 3.61356 4.35482 4.35482C3.61356 5.09608 3.02571 5.97621 2.62491 6.94487C2.22411 7.91353 2.01821 8.9517 2.019 10C2.01821 11.0483 2.22411 12.0865 2.62491 13.0551C3.02571 14.0238 3.61356 14.9039 4.35482 15.6452C5.09608 16.3864 5.97621 16.9743 6.94487 17.3751C7.91353 17.7759 8.9517 17.9818 10 17.981C11.0483 17.9818 12.0865 17.7759 13.0551 17.3751C14.0238 16.9743 14.9039 16.3864 15.6452 15.6452C16.3864 14.9039 16.9743 14.0238 17.3751 13.0551C17.7759 12.0865 17.9818 11.0483 17.981 10C17.9818 8.9517 17.7759 7.91353 17.3751 6.94487C16.9743 5.97621 16.3864 5.09608 15.6452 4.35482C14.9039 3.61356 14.0238 3.02571 13.0551 2.62491C12.0865 2.22411 11.0483 2.01821 10 2.019ZM9 10.117V5C9 4.484 9.5 4 10 4C10.5 4 11 4.5 11 5L11.01 9.6L13.709 12.277C13.912 12.467 14.019 12.726 14.019 12.987C14.019 13.474 13.632 13.951 12.977 13.951C12.707 13.951 12.443 13.862 12.277 13.685L9.329 10.728C9.15345 10.5678 9.03747 10.3527 9 10.118V10.117Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ClockIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M10 0C15.516 0 20 4.484 20 10C20 15.516 15.516 20 10 20C4.484 20 0 15.516 0 10C0 4.484 4.484 0 10 0ZM10 2.019C8.9517 2.01821 7.91353 2.22411 6.94487 2.62491C5.97621 3.02571 5.09608 3.61356 4.35482 4.35482C3.61356 5.09608 3.02571 5.97621 2.62491 6.94487C2.22411 7.91353 2.01821 8.9517 2.019 10C2.01821 11.0483 2.22411 12.0865 2.62491 13.0551C3.02571 14.0238 3.61356 14.9039 4.35482 15.6452C5.09608 16.3864 5.97621 16.9743 6.94487 17.3751C7.91353 17.7759 8.9517 17.9818 10 17.981C11.0483 17.9818 12.0865 17.7759 13.0551 17.3751C14.0238 16.9743 14.9039 16.3864 15.6452 15.6452C16.3864 14.9039 16.9743 14.0238 17.3751 13.0551C17.7759 12.0865 17.9818 11.0483 17.981 10C17.9818 8.9517 17.7759 7.91353 17.3751 6.94487C16.9743 5.97621 16.3864 5.09608 15.6452 4.35482C14.9039 3.61356 14.0238 3.02571 13.0551 2.62491C12.0865 2.22411 11.0483 2.01821 10 2.019ZM9 10.117V5C9 4.484 9.5 4 10 4C10.5 4 11 4.5 11 5L11.01 9.6L13.709 12.277C13.912 12.467 14.019 12.726 14.019 12.987C14.019 13.474 13.632 13.951 12.977 13.951C12.707 13.951 12.443 13.862 12.277 13.685L9.329 10.728C9.15345 10.5678 9.03747 10.3527 9 10.118V10.117Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ClockIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M10 0C15.516 0 20 4.484 20 10C20 15.516 15.516 20 10 20C4.484 20 0 15.516 0 10C0 4.484 4.484 0 10 0ZM10 2.019C8.9517 2.01821 7.91353 2.22411 6.94487 2.62491C5.97621 3.02571 5.09608 3.61356 4.35482 4.35482C3.61356 5.09608 3.02571 5.97621 2.62491 6.94487C2.22411 7.91353 2.01821 8.9517 2.019 10C2.01821 11.0483 2.22411 12.0865 2.62491 13.0551C3.02571 14.0238 3.61356 14.9039 4.35482 15.6452C5.09608 16.3864 5.97621 16.9743 6.94487 17.3751C7.91353 17.7759 8.9517 17.9818 10 17.981C11.0483 17.9818 12.0865 17.7759 13.0551 17.3751C14.0238 16.9743 14.9039 16.3864 15.6452 15.6452C16.3864 14.9039 16.9743 14.0238 17.3751 13.0551C17.7759 12.0865 17.9818 11.0483 17.981 10C17.9818 8.9517 17.7759 7.91353 17.3751 6.94487C16.9743 5.97621 16.3864 5.09608 15.6452 4.35482C14.9039 3.61356 14.0238 3.02571 13.0551 2.62491C12.0865 2.22411 11.0483 2.01821 10 2.019ZM9 10.117V5C9 4.484 9.5 4 10 4C10.5 4 11 4.5 11 5L11.01 9.6L13.709 12.277C13.912 12.467 14.019 12.726 14.019 12.987C14.019 13.474 13.632 13.951 12.977 13.951C12.707 13.951 12.443 13.862 12.277 13.685L9.329 10.728C9.15345 10.5678 9.03747 10.3527 9 10.118V10.117Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ClockIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M10 0C15.516 0 20 4.484 20 10C20 15.516 15.516 20 10 20C4.484 20 0 15.516 0 10C0 4.484 4.484 0 10 0ZM10 2.019C8.9517 2.01821 7.91353 2.22411 6.94487 2.62491C5.97621 3.02571 5.09608 3.61356 4.35482 4.35482C3.61356 5.09608 3.02571 5.97621 2.62491 6.94487C2.22411 7.91353 2.01821 8.9517 2.019 10C2.01821 11.0483 2.22411 12.0865 2.62491 13.0551C3.02571 14.0238 3.61356 14.9039 4.35482 15.6452C5.09608 16.3864 5.97621 16.9743 6.94487 17.3751C7.91353 17.7759 8.9517 17.9818 10 17.981C11.0483 17.9818 12.0865 17.7759 13.0551 17.3751C14.0238 16.9743 14.9039 16.3864 15.6452 15.6452C16.3864 14.9039 16.9743 14.0238 17.3751 13.0551C17.7759 12.0865 17.9818 11.0483 17.981 10C17.9818 8.9517 17.7759 7.91353 17.3751 6.94487C16.9743 5.97621 16.3864 5.09608 15.6452 4.35482C14.9039 3.61356 14.0238 3.02571 13.0551 2.62491C12.0865 2.22411 11.0483 2.01821 10 2.019ZM9 10.117V5C9 4.484 9.5 4 10 4C10.5 4 11 4.5 11 5L11.01 9.6L13.709 12.277C13.912 12.467 14.019 12.726 14.019 12.987C14.019 13.474 13.632 13.951 12.977 13.951C12.707 13.951 12.443 13.862 12.277 13.685L9.329 10.728C9.15345 10.5678 9.03747 10.3527 9 10.118V10.117Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
